storage/rocksdb/ha_rocksdb.cc

```diff
@@ -1157,8 +1148,43 @@ static MYSQL_SYSVAR_BOOL(pause_background_work, rocksdb_pause_background_work,
 
 static MYSQL_SYSVAR_BOOL(
     enable_ttl, rocksdb_enable_ttl, PLUGIN_VAR_RQCMDARG,
-    "Disable expired ttl records from being dropped during compaction.",
-    nullptr, rocksdb_set_enable_ttl, TRUE);
+    "Enable expired TTL records to be dropped during compaction.", nullptr,
+    nullptr, TRUE);
+
+static MYSQL_SYSVAR_BOOL(
```

Comment:
yes, I considered it. Hiding them completely would break `main.mysqld-help-notwin` test.  It will also break `rocksdb.rocksdb` test.  This is why I left them there.  I do explicitly say in the variable description that it is a no-op in non-debug builds.